templates/contact.html

```diff
@@ -57,6 +57,20 @@
   code.discord {
     cursor: pointer;
   }
+
+  @media screen and (max-width: 487px) {
```

Comment:
```suggestion
  @media screen and (max-width: 487px) {
```
Best to use bootstrap breakpoints for consistency (ensures layouts break at the same point)

https://getbootstrap.com/docs/5.3/layout/breakpoints/#media-queries